src/store/plugins/names.js

```diff
@@ -105,19 +105,22 @@ export default (store) => {
         const names = await Promise.all(
           accounts.map(({ address }) => Promise.all([
             getPendingNameClaimTransactions(address),
-            middleware.getNamesOwnedBy(address)
-              .then(({ active }) => active.map(({ info, name, hash }) => ({
-                createdAtHeight: info.activeFrom,
-                expiresAt: info.expireHeight,
-                owner: info.ownership.current,
-                pointers: info.pointers,
-                autoExtend: !!(
-                  owned.find((n) => n.name === name)?.autoExtend
-                  || pendingAutoExtendNames?.includes(name)
-                ),
-                name,
-                hash,
-              }))),
+            middleware.getNames({ owned_by: address })
+              .then(({ data }) => data
+                .filter(({ active }) => active)
+                .map(({ info, name, hash }) => ({
+                  createdAtHeight: info.activeFrom,
+                  expiresAt: info.expireHeight,
+                  owner: info.ownership.current,
+                  pointers: info.pointers,
+                  autoExtend:
+                  !!(
```

Comment:
I don't see a reason to move this into new line. I think the way how it was before was OK.